modules/embeddingsmodel/src/main/kotlin/com/theagilemonkeys/ellmental/embeddingsmodel/openai/OpenAIClient.kt

```diff
@@ -0,0 +1,33 @@
+package com.theagilemonkeys.ellmental.embeddingsmodel.openai
+
+import com.aallam.openai.api.http.Timeout
+import com.aallam.openai.client.*
+import kotlin.time.Duration.Companion.seconds
+
+/**
+ * # OpenAI client generator
+ *
+ * Wrapper to create an [OpenAI] client with default parameters for JVM compatibility
+ *
+ */
+@JvmOverloads fun openAIClient(
```

Comment:
MINOR SUGGESTION

I would either rename it as a get function or "fake" it as if you were using an instance of a class (`OpenAiClient`)

```suggestion
@JvmOverloads fun getOpenAIClient(
```

```suggestion
@JvmOverloads fun OpenAIClient(
```

Reply:
Thanks! <3